shared/js/initApp.js

```diff
@@ -302,8 +302,11 @@ if (appOptions.droplet) {
   promise = loadSource('jsinterpreter/acorn_interpreter')()
       .then(loadSource('requirejs/require'))
       .then(loadSource('ace/ace'))
+      .then(loadSource('ace/mode-javascript'))
+      // .then(loadSource('applab/mode-javascript_codeorg'))
       .then(loadSource('ace/ext-language_tools'))
-      .then(loadSource('droplet/droplet-full.min'));
+      .then(loadSource('droplet/droplet-full'));
```

Comment:
Most of these files we seem to generate one of the same name whether we're minified or not. Droplet-full does not seem to do that. We'd probably like it to, though that's a separate issue.